apps/src/lib/cli/context.rs

```diff
@@ -166,7 +166,7 @@ impl Context {
 
     /// Read the given WASM file from the WASM directory or an absolute path.
     pub fn read_wasm(&self, file_name: impl AsRef<Path>) -> Vec<u8> {
-        wasm_loader::read_wasm(self.wasm_dir(), file_name)
```

Comment:
I think in here we could turn it into a more user-friendly error instead of panic

Reply:
There are 20 calls to this method so for now I've changed it back to the previous behaviour here (`eprintln!` + `safe_exit`), `make test-unit` still finishes so guess it wasn't a call to this that was causing it to terminate